java-symbolic-execution/src/test/files/se/FlowComputationRelSV.java

```diff
@@ -58,7 +58,6 @@ void npe1(Object a) {
 
   void npe2(Object a, Object b) {
     if (a == b) {
-      // FIXME SONARJAVA-2272
```

Comment:
The situation is not fixed, it should refer to https://jira.sonarsource.com/browse/SONARJAVA-2303 now.

Reply:
missed this while reading SONARJASVA-2272